uisdk/src/androidTest/java/com/karhoo/uisdk/trip/TripTests.kt

```diff
@@ -333,7 +333,7 @@ class TripTests : Launch {
         mockTripSuccessResponse(
                 status = TRIP_STATUS_ARRIVED,
                 tracking = DRIVER_TRACKING,
-                details = TRIP_ARRIVED)
+                details = TRIP_DER_NO_VEHICLE_AND_DRIVER_DETAILS)
```

Comment:
Is it the test that is wrong or the code for this? Should the user be able to cancel once the driver has arrived?

Reply:
This is the test which is covering the latest changes. If I am not mistaken when the Trip status changes to the TRIP_ARRIVED we hide the Cancel button in this screen. therefore the test is in valid and wrong especially lines [348](https://github.com/karhoo/karhoo-android-ui-sdk/pull/104/files#diff-eafb9a51edd057d51a24ed370cee7a471c653859165d3e3698d800ff3648464fR348) and [349](https://github.com/karhoo/karhoo-android-ui-sdk/pull/104/files#diff-eafb9a51edd057d51a24ed370cee7a471c653859165d3e3698d800ff3648464fR349)